docker/weaviate/docker-compose.yml

```diff
@@ -10,9 +10,11 @@ services:
       QUERY_DEFAULTS_LIMIT: 25
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
       PERSISTENCE_DATA_PATH: '/var/lib/weaviate'
-      DEFAULT_VECTORIZER_MODULE: 'none'
-      ENABLE_MODULES: 'text2vec-gpt4all'
-      GPT4ALL_INFERENCE_API: 'http://text2vec-gpt4all:8080'
+      ENABLE_MODULES: text2vec-transformers
```

Comment:
I changed these to be more general. The GPT4all transformers aren't necessary because the bindings in the vecdb don't ever interact with bindings in an LLM